### ISSUE TITLE
Consider using a different jQuery multiselect plugin.

### ISSUE THREAD

**Issue by jrwdunham** [jrwdunham/dative#101]
The current multiSelect one seems to be buggy:
- can't shift+tab reverse out of it
- scroll bars don't appear consistently
- takes up a lot of space, with no easy way of collapsing it...

Maybe use https://github.com/ehynds/jquery-ui-multiselect-widget

**Comment by jrwdunham** [comment]
Or use magicsuggest for OLD tags as is the plan for FieldDB tags (cf. #98).

**Comment by cesine** [comment]
louisa is shopping for a typeahead library so we wanted to know your feedback about what josh found a few years ago.

how do you add new options in https://github.com/ehynds/jquery-ui-multiselect-widget 

that makes it not a `typeahead` but a `dropdown` no? as a design principle for usability we only want typeaheads so the user doesn't have to go to a separate part of the app to add new options

**Comment by jrwdunham** [comment]
@cesine @louisa-bielig: I have switched over to using Tag-It (https://github.com/aehlke/tag-it) for this. It seems to work reasonably well. I have created a `MultiElementTagFieldView` for this and it is being used in Dative to allow users to select existing tags and create new ones without leaving the interface. Right now, it asks the user to confirm that they want to create a new tag, but that could be made optional, if we need...
